app/views/content_items/detailed_guide.html.erb

```diff
@@ -6,15 +6,26 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render 'govuk_publishing_components/components/title', @content_item.title_and_context %>
+    <% title_and_context = @content_item.brexit_hub_page? ? { title: @content_item.title } : @content_item.title_and_context %>
+    <%= render 'govuk_publishing_components/components/title', title_and_context %>
   </div>
   <%= render 'shared/translations' %>
   <div class="govuk-grid-column-two-thirds">
-    <%= render 'govuk_publishing_components/components/lead_paragraph', text: @content_item.description %>
+    <% if @content_item.brexit_hub_page? %>
+      <div class="govuk-body-l govuk-!-margin-bottom-7">
+        <% link_text = @content_item.brexit_citizen_page? ? "Brexit guidance for businesses" : "Brexit guidance for individuals and families" %>
```

Comment:
It feels like this text might belong in the same helper - or even better in a locale file?

Having something like one of these in the helper:
```
brexit_links = { 
                        <BUSINESS_PAGE_CONTENT_ID>: {
                           text: "Brexit guide for businesses",
                           url: "/url"
                        },
                       <CITIZEN_PAGE_CONTENT_ID>: {
                           text: t(brexit.citizen_page.link_text),  # Brexit guide for individuals etc.
                           url: t(brexit.citizen_page.link_path) # /guidance/brexit-stuff
                        }

def brexit_link
   brexit_links[content_item["content_id"]]
end
```

That would let you flatten this out a bit to:

```
<% if brexit_link %>
  <p class="govuk-body">
  There's different <%= link_to brexit_link[:text], brexit_link[:path] %>
  </p>
<% end %>
```

Reply:
Roger. Thanks